Update the Two-Factor Authentication page information (#1682)
magento/merchdocs: src/stores/security-two-factor-authentication.md

{: .bs-callout-info}
This implementation of two-factor authentication (2FA) applies to the _Admin_ only, and is not available for customer accounts. The two-factor authentication that protects your [magento.com account]({% link magento/magento-account.md %}) has a separate setup. To learn more, go to [Securing Your Account]({% link magento/magento-account-secure.md %}).

> Two-factor authentication is widely used, and it is common to generate access codes for different websites on the same app. This ensures that only you is able to log in. If you lose your password or a bot guesses it, two-factor authentication will be the last layer of protection. For example, you might use Google Authenticator to generate codes for the _Admin_ of your store, your _magento.com_ account, Google account, and so on.

Reviewer: ```suggestion
Two-factor authentication is widely used, and it is common to generate access codes for different websites on the same app. This ensures that only you are able to log in to your user account. If you lose your password or a bot guesses it, two-factor authentication adds a layer of protection. For example, you might use Google Authenticator to generate codes for the _Admin_ of your store, your _Commerce account_ account, Google account, and so on.
```

1. In the _General_ section, select each **Provider to use**.

   |--- |--- |
   |Google Authenticator|Generates a one-time password in the application for user authentication.|
   |Duo Security|Provides SMS and push notification.|
   |Authy|Generates a time-dependent six-digit code and delivers SMS or Voice Call 2FA protection or token.|

>    |U2F Devices (Yubikey and others)| Physical device to authenticate, like [YubiKey](https://www.yubico.com/).|

Reviewer: ```suggestion
   |U2F Devices (Yubikey and others)| Uses a physical device to authenticate, such as [YubiKey](https://www.yubico.com/).|
```
